content/news/release-20180218.md

```diff
@@ -0,0 +1,56 @@
+We are excited to announce the first [OpenRA release](/download/) of this year!
+Over 700 commits from 43 contributors were merged over the last six months, with each commit helping to fix bugs, improve code quality, or add new features that improve the single- and multi-player experience.
+
+The main focus of this development cycle was modernizing core parts of the engine and game code to support the work-in-progress Tiberian Sun mod and other community projects. Thanks to all bug reporters and contributors who helped with this task! Please note that changes to the way hotkeys are managed mean you will need to redefine any custom hotkeys in the Input settings.
```

Comment:
The second sentence doesn't flow from the first.  How about "Thanks to our contributors, and to everybody who tested the playtest and reported bugs!"